tsec-http4s/src/test/scala/tsec/authentication/RequestAuthenticatorSpec.scala

```diff
@@ -189,7 +189,7 @@ class RequestAuthenticatorSpec extends AuthenticatorSpec {
 
       response.getOrElse(Response.notFound).map(_.status).unsafeRunSync() mustBe Status.BadGateway
     }
-
+jw
```

Comment:
Por que?

Reply:
laptop fat finger. shush it never happened